docs/book/src/reference/good-practices.md

```diff
@@ -0,0 +1,24 @@
+# Good Practices
+
+## What is "Reconciliation" in Operators?
+
+The reconciliation is a continuos  loop that performs necessary actions on the current state to ensure that the Custom Resource reaches the desired state as specified by the user.
+
+## Why should reconciliations be idempotent?
```

Comment:
Do the questions fit well for a good practices doc? We might want to add the golden rules to provide the recommendations and their explanation directly, like the Operator-sdk doc ( https://sdk.operatorframework.io/docs/best-practices/common-recommendation/).